src/helpers/cliUtils.js

```diff
@@ -128,6 +129,43 @@ async function extractDataForPatients(auth, config, patientIds, icareClient, mes
   return errors;
 }
 
+async function sendEmailNotification(notificationInfo, errors) {
+  const totalErrors = Object.keys(errors).reduce((previousValue, currentValue) => previousValue + errors[currentValue].length, 0);
+  if (totalErrors === 0) {
+    return;
+  }
+
+  if (!notificationInfo.from || !notificationInfo.to || !notificationInfo.host || !notificationInfo.port) {
+    throw new Error('Notification information incomplete. Unable to send email.');
```

Comment:
DP2 - Could we provide a message here that points users to where this could be update this information, and that there were errors that weren't sent out? Candidate message: 
`Email notification information incomplete, unabel to send email containing ${totalErrors} errors. Update the notificationInfo configuration to receive emails when errors occur`

Reply:
Yeah that's a good idea!